clients/extension/src/utils/api.ts

```diff
@@ -1,3 +1,4 @@
+import { SolanaJupiterToken } from '@clients/desktop/src/coin/jupiter/token'
```

Comment:
Not a good idea to import `clients/desktop`

Reply:
Updated